components/TextArea.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+const TextArea = ({
+  label,
+  name,
+  onChange,
+  required,
+  minLength,
+  maxLength,
+  width,
+  height,
+  padding,
+  margin,
+}) => (
+  <div className="input-root">
+    <label htmlFor={name}>{label}</label>
+    <textarea
+      name={name}
+      onChange={onChange}
+      required={required}
+      minLength={minLength}
+      maxLength={maxLength}
+    />
+    <style jsx>{`
+      .input-root {
+        display: flex;
+        flex-flow: column;
+        max-width: 300px;
+        margin: 10px auto;
+      }
+
+      label {
+        font-size: 12px;
+        text-transform: uppercase;
+        display: block;
+        font-family: 'Source Sans Pro', sans-serif;
```

Comment:
You don't have to define it when it has been defined on the body element

Reply:
was used for storybook, now problem solved